gitlab/resource_gitlab_group.go

```diff
@@ -61,9 +61,51 @@ func resourceGitlabGroup() *schema.Resource {
 			"visibility_level": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				Computed:     true,
+				Default:      "private",
```

Comment:
What's the reason for changing the default?

Reply:
The gitlab api doesn't require this field.  When not provided, the api will provide a default, but it varies depending on what version of gitlab you're using.  Adding a default here forces deterministic behavior.

Reply:
Cool, thanks.

Reply:
This is a breaking change, I don't think we need that?

Reply:
@roidelapluie Do you want me to revert it, or is our next release a major one anyway? The reason I was ok with this is because the provider documentation for this resource already says "private" is the default.

Reply:
In general I think we should have as much computed as possible, and fix the doc if it is not true?

Reply:
I started disliking O+C attributes after reading through [this issue](https://github.com/hashicorp/terraform/issues/21278), if you're interested.

Reverting the change to this attribute and fixing up the docs makes sense. I can do a PR later.

Reply:
But you can remove the  O ?

Reply:
Removing the O would make it no longer an argument, right? We still want it to be an argument that can be set.

Reply:
mmh I am not 100% sure. But we can keep the O, we do that elsewhere.

Reply:
PR for this: #408